src/sentry/api/endpoints/project_create_sample.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases.project import ProjectEndpoint, ProjectPermission
+from sentry.api.serializers import serialize
+from sentry.utils.samples import create_sample_event
+
+
+class RelaxedProjectPermission(ProjectPermission):
+    scope_map = {
+        'POST': ['project:write', 'project:admin'],
```

Comment:
it looks like this has the same permissions level as `ProjectPermission`, so why is this necessary?